content/en/synthetics/browser_tests/_index.md

```diff
@@ -0,0 +1,112 @@
+5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. The full list is retrievable through the [Datadog API][2]. You can also set up a [Private Location][3] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Intervals are available between every 15 minutes to once per week. [Contact support][4] to enable additional frequencies for your test.
+
+### Use global variables
+
+You can use the [global variables defined in the `Settings`][5] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
+
+{{< img src="synthetics/browser_tests/using_variables_browser.mp4" alt="Using Variables in Browser Tests" video="true"  width="80%" >}}
+
+### Alert conditions
+
+An alert is triggered if any assertion fails for `<INSERT_NUMBER>` minutes from any `<INSERT_NUMBER>` of `<NUMBER_OF_CHOSEN>` locations.
+
+### Notifications
+
+To configure your notifications:
+
+1. Enter a **message** for the browser test. This field allows standard [Markdown formatting][6]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
+2. Choose your [services][7] and/or team members to notify.
+3. Click **Save Details and Record Test** to save your browser test.
+4. Start to record your test.
+
+## Record test
+
+Tests can be only recorded from **[Google Chrome][8]**. To record your test, download the [Datadog Record Test extension for Google Chrome][9].
+
+{{< img src="synthetics/browser_tests/browser_check_record_test.png" alt="Browser test record test"  >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+Hitting **Open in a pop-up** and **Shift** will open the pop up in incognito mode. This is useful to start recording your test from a fresh browser free from already logged in sessions, cookies from your existing browser, etc.
+2. Click on **Start recording** to begin recording your browser test.
+3. Your actions are recorded and used to create steps within your browser test scenario.
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_tests/browser_test_step.png" alt="Browser Test steps"  style="width:80%;">}}
+
+    **Note**: **Your last browser test step must be an Assertion**, otherwise there is nothing to check.
+5. Once you have finished your Scenario, click on **Save and Launch Test**.
+
+## Actions
+
+After saving a Browser Test you can record [Actions][10] as a series of steps, which you can then edit or build on. Actions can also be configured with [advance settings][11].
```

Comment:
```suggestion
After saving a Browser Test you can record [Actions][10] as a series of steps, which you can then edit or build on. Actions can also be configured with [advanced settings][11].
```

I think?

Reply:
I'm not seeing this sentence, but I'm seeing all the actions on the general browser test page - is it just a cache issue?

Reply:
Nope, definitely was a me issue - sorry! I ended up pushing the commit without removing from the original file. Will remove this and the other sections we talked about removing.

Reply:
This is odd. It's not in my file locally or here, but is showing up still on the staging page. @l0k0ms can you take a look into this?

Reply:
@sarina-dd fixed it, the old file wasn't deleted and the aliases were doing a circular dependency. This should be fixed now :)